### DISCUSSION TITLE
Is stablecoin appropriate as reward?

### DISCUSSION THREAD

**Discussion by tktkowash** [JasmyCoin/JasmyCoin#13] [General]
Stablecoin, ERC-20 token, or any other crypto asset, which is better as reward?

**Comment by harayamada** [comment]
For whom?

**Reply by harayamada** [reply]
The issue of a "stablecoin" is that it is simply pegged to fiat or that its price is stable, as some other superficial aspect. However, as the issuer of the currency, why do they issue it? And why is it necessary to stabilize the price?

**Reply by tktkowash** [reply]
Generally speaking, they issue a currency to make their market, control economy, or get seigniorage. Currency stability is important to make the currency more dominant.

As for fiat currency, several institutions, such as central bank or IMF, are trying to stabilize the price of the currency. However, since no one can't control the price of crypto assets, pegging to fiat currency is better way to stabilize the price.

**Reply by harayamada** [reply]
For trading that includes rewards in the real-life economy market, it is desirable to have a robust currency that users can use with confidence to support the market,personal data market too^_^

**Reply by toktokds01** [reply]
Stable coins are necessary, but rewards do not necessarily have to be stable coins.
Some people want volatility, so it depends.

**Reply by harayamada** [reply]
Yes!ｍｍｍ・・・ Who would want volatility…?
What kind of value should rewards be tied to…?
